src/components/ImageComponent/ImageComponent.js

```diff
@@ -0,0 +1,11 @@
+import { observerListener } from '../../shared/observer';
+
+const template = (imageUrl, className, id) => {
```

Comment:
I think we could connect that to the TileComponent

There is task to add onClick to tileComponent, if you can please update this task to use just this image component and that should do the trick

Reply:
Done